spec/shr_base_finding.txt

```diff
@@ -329,17 +329,10 @@ Description:	"An entry concerning a patient where the result is a narrative text
 1..1			Patient
 //1..1			CommentOrDescription // put into DataValue
 
+
 EntryElement:	BodyStructurePresent
 Based on:		Condition
-Description:	"A body structure such as wound or tumor, observed to be present in a patient. The body structure is a condition that persists over time. The Code will represent the body structure, and the BodySiteCode the location of that structure. This corresponds to FHIR R4, where the BodyStructure resource has morphology (corresponding to Code) and location (corresponding to BodySite.BodySiteCode). For example, the Code could be blunt force injury (SCT 3821009) and location left knee."
-1..1			Patient
-0..1			BodySite
-0..1			Morphology
-0..1			RelevantTime
+Description:	"A body structure such as wound or tumor, observed to be present in a patient. The body structure is a condition that persists over time. The Code represents the body structure itself (the morphology), while the BodySiteCode is the location of that structure. This corresponds to FHIR R4, where the BodyStructure resource has morphology (corresponding to Code) and location (corresponding to BodySite.BodySiteCode. For example, the Code could be blunt force injury (SCT 3821009) and location left knee."
+				Code from BodyStructureVS if covered
```

Comment:
Can't we use http://hl7.org/fhir/ValueSet/body-site instead of creating our own value set?

Reply:
There is a slight difference. FHIR body-site excludes a number of branches under body structure, including "morphologically altered structure" -- which would seem to be important to include. Don't you think?

Reply:
You're right, of course!